pkgs/top-level/ocaml-packages.nix

```diff
@@ -297,10 +299,14 @@ let
 
     ocamlfuse = callPackage ../development/ocaml-modules/ocamlfuse { };
 
+    ocaml_gettext = callPackage ../development/ocaml-modules/ocaml-gettext { };
```

Comment:
In https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/ocaml-packages.nix there are dozen attributes starting with ```ocaml_``` and zero starting with ```ocaml-```. ```ocaml-``` may be better but shouldn't other names in the file be changed as well?